internal/performance/helper.go

```diff
@@ -55,7 +56,7 @@ func setup(tb testing.TB, dev bool) (*quickstore.Quickstore, func(context.Contex
 	defer cancel()
 
 	benchmarkConfig := &mpb.BenchmarkInfo{}
-	if dev {
+	if os.Getenv("PERFORMANCE_DEV") == "1" {
```

Comment:
Can this be part of config as line 45 above, the env vars will be processed by github.com/sethvargo/go-envconfig

Reply:
cool, thanks. Trying it out.

Reply:
Done, I made a new config for both Mako and the tests, in the event that we want to tune more params in the future.